metadata.rb

```diff
@@ -10,6 +10,7 @@
 
 supports         'centos', '~> 6.0'
 supports         'centos', '~> 7.0'
+supports         'debian'
```

Comment:
Why is debian being added as a platform we support here?

Edit: I see the reference in the apc recipe. We can just go ahead and remove all Debian platform references as we no longer need to support it.